onnxscript/tests/function_libs/torch_aten/extra_opinfo.py

```diff
@@ -211,23 +211,23 @@ def sample_inputs_layer_norm(
         supports_out=False,
     ),
     opinfo_core.OpInfo(
-        "nn.functional.conv3d",
```

Comment:
The diff in this file seems just swapping the test upside down? Should be able to revert since there is no real change right?

Reply:
yes :)